projects/aca-content/src/lib/components/sidenav/components/sidenav-header.component.html

```diff
@@ -4,10 +4,9 @@
       tabindex=0
       (click)="toggleNavBar.emit()"
       (keypress)="toggleNavBar.emit()">
-      <img
```

Comment:
you are killing the existing feature to display custom logo that customers can change

Reply:
![image](https://github.com/Alfresco/alfresco-content-app/assets/112551534/50d5f1c1-0c45-4a94-af42-61e4416ade0a)
It is a change suggested in https://alfresco.atlassian.net/browse/ACS-5166. To replace the logo with collapse icon.

Reply:
if this is approved, you need to cleanup the code that used to do the logo$ and everything related to that logo and not leave unused code in the repo

Reply:
hi @DenysVuika , there was a miscommunication, which caused this change. We have made the right changes now. you can please review the PR